subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -269,12 +285,14 @@ static int job_update_rejected(struct mqtt_client *const client,
 			       u32_t payload_len)
 {
 	LOG_ERR("Job document update was rejected");
+	execution_version_number--;
```

Comment:
Why do we need this?

Reply:
Error handling, we increment when we send an update. If the update is rejected we need to decrement the number back to what it was.